Plugins/Triage/Retrieve-Items.ps1

```diff
@@ -63,20 +63,36 @@ process{
     $7z64bitTestPath = Get-Item -Path $7za64 -ErrorAction SilentlyContinue
     $7z32bitTestPath = Get-Item -Path $7za32 -ErrorAction SilentlyContinue
 
-    if (-not $7z64bitTestPath) {
+    if (!$7z64bitTestPath) {
 
         Throw "64 bit version of 7za.exe not detected in Bin. Place 64bit executable in Bin directory and try again."
 
-    } elseif (-not $7z32bitTestPath) {
+    } elseif (!$7z32bitTestPath) {
 
         Throw "32 bit version of 7za.exe not detected in Bin. Place 32bit executable in Bin directory and try again."
     }
 
+    #Verify that Velociraptor executables are located in $global:PowerREsponse.Config.Path.Bin (For locked files)
+
+    $Velo_64 = ("{0}\Velociraptor-amd64.exe" -f $global:PowerResponse.Config.Path.Bin)
+    $Velo_32 = ("{0}\Velociraptor-386.exe" -f $global:PowerResponse.Config.Path.Bin)
+
+
+    if (!$Velo_64) {
```

Comment:
These will always resolve to `True` since you're just checking the existence of the string `$Velo_64|32`.

One way to check multiple paths like this would be to set `$7za64`,`$7za32`,`$Velo_64`,`$Velo_32` as normal, then:
```
$MissingExe = '$7za64','$7za32','$Velo_64','$Velo_32' | Where-Object { !(Test-Path -Path $PSItem) }
if ($MissingExe) {
    Throw ('Missing dependency detected: {0}' -f ($MissingExe -Join ', '))
}
```

Reply:
fixed.